code/modules/clothing/outfits/plasmaman.dm

```diff
@@ -21,7 +21,7 @@
 
 	uniform = /obj/item/clothing/under/plasmaman/enviroslacks
 	gloves = /obj/item/clothing/gloves/color/plasmaman/white
-	head = /obj/item/clothing/head/helmet/space/plasmaman/white
+	head = /obj/item/clothing/head/helmet/space/plasmaman/detective
```

Comment:
```suggestion
	head = /obj/item/clothing/head/helmet/space/plasmaman/detective
```

Please use comments to mark lines of upstream code that you change (here and in two other places in this file). Otherwise this looks good to me.